### DISCUSSION TITLE
example of creating ODFs using jMakeOdf

### DISCUSSION THREAD

**Discussion by davidgritter** [GrandOrgue/grandorgue#1140] [General]
I recently made a GrandOrgue ODF for the recently released free Buckeburg organ from Sonus Paradisi, so I thought I would document how to do this using  Lars Palo's jMakeOdf java program.  I put this on a web page where you can also download the input file and the final odf. 
its at:
https://diyorganconsole.wordpress.com/creating-grandorgue-organ-files-from-hauptwerk-samplesets/

**Comment by thbeckpr** [comment]
Thanks for this. However, on loading the ODF this error -and 172 others in the trem directory- is thrown:

Fri 24 Jun 2022 09:11:22 AM AST: 09:11:22 AM: Error: File 'pipes\Buckeburg_3noc_OW\OW_Principal8\03-AB_OW_Principal8\trem\036-C.wav' does not exists

Also, not to steal your thunder, but an ODF for this great organ has been created by an individual in the UK. It is derived from the HW ODF and includes the graphics and I believe as much of the HW functionality as GO supports, including a mixer panel to adjust the channel balance.

He has other ODFs available here:
https://sites.google.com/view/andrews-odfs/home

**Reply by davidgritter** [reply]
Normally errors such as this result when you have not deleted the trem directories  in the original buckeburg distribution  and replaced them with symbolic links as described in the text of the web page I linked to in the original message.  The trem directories in the original distribution contain samples for hauptwerk synthesized trems, but to use jMakeOdf to create the .organ file these need to be deleted and replaced by links to the actual tremmed sample directory
